Core/src/org/sleuthkit/autopsy/directorytree/Bundle.properties

```diff
@@ -124,3 +117,16 @@ GroupDataSourcesDialog.yesButton.text=Yes
 GroupDataSourcesDialog.noButton.text=No
 GroupDataSourcesDialog.title=Group by Data Source?
 DirectoryTreeTopComponent.openViewPreferencesButton.text=
+ExternalViewerGlobalSettingsPanel.jTable1.columnModel.title3=Title 4
+ExternalViewerGlobalSettingsPanel.jTable1.columnModel.title2=Title 3
+ExternalViewerGlobalSettingsPanel.jTable1.columnModel.title3_1=Title 4
+ExternalViewerGlobalSettingsPanel.jTable1.columnModel.title2_1=Title 3
+ExternalViewerGlobalSettingsPanel.newRuleButton1.text=New Rule
+ExternalViewerGlobalSettingsPanel.editRuleButton.text=Edit Rule
+ExternalViewerGlobalSettingsPanel.deleteRuleButton.text=Delete Rule
+ExternalViewerGlobalSettingsPanel.externalViewerTitleLabel.text=Add your custom rules for external viewers:
+ExternalViewerGlobalSettingsPanel.newRuleButton.text=New Rule
+ExternalViewerGlobalSettingsPanel.jButton2.text=jButton2
+ExternalViewerGlobalSettingsPanel.browseHxDDirectory.text=Browse
+ExternalViewerGlobalSettingsPanel.HxDLabel.text=HxD Editor Path:
+ExternalViewerGlobalSettingsPanel.ContentViewerExtensionLabel.text=Add content viewer extensions:
```

Comment:
@dannysmyda and @APriestman, did we remove any deleted items from the Japanese bundle?

Reply:
For some reason, there is a desync between the English and Japanese bundles. No ExternalViewerGlobalSettingsPanel or ExternalRulePanel (which was a feature present before my work) are present in the Japanese bundle.